recipes/paraview/5.13.2/scripts/job-build-daint.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
```

Comment:
Just curious, is this your personal helper script for building the uenv, or is this expected to end up in the uenv? Can you keep them outside of the recipe?

Reply:
Keeping them with the recipe makes it easier to manage versions, since the scripts might change slightly with different releases (version stamps usually etc). Does it do any harm to have them in the repo, they serve as useful tools for other people who want to experiment with the builds.

Reply:
I agree with @msimberg on this.


My 2cents

> Does it do any harm to have them in the repo?

We might end up with tens of different scripts (there are already a good amount to support the point), one per system, that no one will maintain and probably use because they would have to study/change/update them first. So, maybe they look unharmful, but they already are harmful to the cleanliness of the recipe.